### PR TITLE
What's new 8.13

### PR DESCRIPTION
**Pull request by natasha-moore-elastic**
Resolves https://github.com/elastic/security-docs/issues/4808.

Preview: [What’s new in 8.13](https://security-docs_bk_4979.docs-preview.app.elstc.co/guide/en/security/master/whats-new.html)

### CODE REVIEW

File: docs/whats-new.asciidoc [hunk 1]
```diff
@@ -4,106 +4,138 @@
...
 [role="screenshot"]
-image::whats-new/images/8.12/alert-assigned-alerts.png[Alert assignees in the Alerts table,80%]
+image::whats-new/images/8.13/alert-details-flyout-right-panel.png[Right panel of the alert details flyout, 75%]
 
 [float]
-== Timeline enhancements
+== Persistence of Data Quality dashboard results 
 
-The following enhancements have been added to Timeline:
+The {security-guide}/data-quality-dash.html[Data Quality dashboard] now retains results across sessions, ensuring continuity of information. Additionally, the dashboard now shows timestamps indicating when each index was last checked.
```
Comment (benironside) [inline docs/whats-new.asciidoc:43]:
```suggestion
The {security-guide}/data-quality-dash.html[Data Quality dashboard] now retains results across sessions, ensuring continuity of information. Additionally, the dashboard now shows when each index was last checked.
```
simpler / more accurate this way

File: docs/whats-new.asciidoc [hunk 2]
```diff
@@ -4,106 +4,138 @@
...
 [role="screenshot"]
-image::whats-new/images/8.12/exclude-cold-frozen-tiers.png[Advanced setting to exclude cold and frozen tiers from analyzer queries,80%]
+image::whats-new/images/8.13/automated-response-actions.png[Automated response actions, 85%]
 
 [float]
-== Bidirectional integration response actions (SentinelOne)
+=== Third-party response actions (SentinelOne)
 
-Powered by the {integrations-docs}/sentinel_one[SentinelOne] integration for {agent}, SentinelOne response actions allow you to perform bidirectional actions on protected hosts, such as directing SentinelOne to isolate a suspicious endpoint from your network, without needing to leave the {elastic-sec} UI.
+{security-guide}/third-party-actions.html#sentinelone-response-actions[Direct SentinelOne] to perform response actions on protected hosts without leaving the {elastic-sec} UI. You can isolate and release a host from detection alerts and the response console, and view third-party actions in the response actions history log.
```
Comment (benironside) [inline docs/whats-new.asciidoc:84]:
```suggestion
You can now {security-guide}/third-party-actions.html#sentinelone-response-actions[direct SentinelOne] to perform response actions on protected hosts without leaving the {elastic-sec} UI. You can isolate and release a host from detection alerts and the response console, and view third-party actions in the response actions history log.
```

File: docs/whats-new.asciidoc [hunk 3]
```diff
@@ -4,106 +4,138 @@
...
 
-You can now use `matches` and `does not match` conditions on more fields when configuring {security-guide}/event-filters.html[event filters] and {security-guide}/add-exceptions.html[endpoint exceptions]. Previously, only the `file.path.text` field was supported.
+The following enhancements have been added to Entity Analytics:
 
 [float]
-== Cloud Security enhancements
+=== Asset criticality
 
-The following enhancements have been added to Cloud Security:
+You can now assign an {security-guide}/asset-criticality.html[asset criticality] level to your entities, based on the impact they have on your organization. For example, you can assign **Extreme impact** to business-critical entities, or **Low impact** to entities that pose minimal risk to your security posture.
```
Comment (benironside) [inline docs/whats-new.asciidoc:94]:
```suggestion
You can now assign an {security-guide}/asset-criticality.html[asset criticality] level to your entities, based on their importance to your organization. For example, you can assign **Extreme impact** to business-critical entities, or **Low impact** to entities that pose minimal risk to your security posture.
```

File: docs/whats-new.asciidoc [hunk 4]
```diff
@@ -4,106 +4,138 @@
...
+
+The risk scoring engine includes asset criticality as a risk input when calculating entity risk scores.
```
Comment (benironside) [inline docs/whats-new.asciidoc:96]:
```suggestion
The risk scoring engine includes asset criticality as an input when calculating entity risk scores.
```

**Review by benironside — COMMENTED** [review]
Nice work, good screenshot selections! Left a few minor suggestions for your consideration

File: docs/whats-new.asciidoc [hunk 5]
```diff
@@ -4,106 +4,138 @@
...
+=== Cloud native vulnerability management (CNVM) Findings UI enhancements
+
+The **Vulnerabilities** table on the {security-guide}/vuln-management-findings.html[Findings page] now includes improved grouping capabilities (up to three nested groupings), and more table customization options.
+
+image::whats-new/images/8.13/cnvm-findings-grouped.png[CNVM findings grouped, 85%]
+
+[float]
+== Custom fields for cases must have a default value
+
+If you {security-guide}/cases-open-manage.html#cases-ui-custom-fields[add a custom field] to a case, and set the field to be required, you cannot leave the field empty. You must set a default value for the field.
```
Comment (nastasha-solomon) [inline docs/whats-new.asciidoc:138]:
```suggestion
If you {security-guide}/cases-open-manage.html#cases-ui-custom-fields[add a custom field] to a case, and set the field to be required, you cannot leave the field empty. You must set a default value for it.
```

**Review by nastasha-solomon — APPROVED** [review]
Great job and thank you for pulling this together!

**Review by joepeeples — DISMISSED** [review]

File: docs/whats-new.asciidoc [hunk 6]
```diff
@@ -4,106 +4,138 @@
...
 
-You can now use `matches` and `does not match` conditions on more fields when configuring {security-guide}/event-filters.html[event filters] and {security-guide}/add-exceptions.html[endpoint exceptions]. Previously, only the `file.path.text` field was supported.
+The following enhancements have been added to Entity Analytics:
 
 [float]
-== Cloud Security enhancements
+=== Asset criticality
 
-The following enhancements have been added to Cloud Security:
+You can now assign an {security-guide}/asset-criticality.html[asset criticality] level to your entities, based on their importance to your organization. For example, you can assign **Extreme impact** to business-critical entities, or **Low impact** to entities that pose minimal risk to your security posture.
```
Comment (jmikell821) [inline docs/whats-new.asciidoc:94]:
```suggestion
You can now assign an {security-guide}/asset-criticality.html[asset criticality] level to your entities based on their importance to your organization. For example, you can assign **Extreme impact** to business-critical entities, or **Low impact** to entities that pose minimal risk to your security posture.
```

File: docs/whats-new.asciidoc [hunk 7]
```diff
@@ -4,106 +4,138 @@
...
+=== Cloud native vulnerability management (CNVM) Findings UI enhancements
+
+The **Vulnerabilities** table on the {security-guide}/vuln-management-findings.html[Findings page] now includes improved grouping capabilities (up to three nested groupings), and more table customization options.
+
+image::whats-new/images/8.13/cnvm-findings-grouped.png[CNVM findings grouped, 85%]
+
+[float]
+== Custom fields for cases must have a default value
+
+If you {security-guide}/cases-open-manage.html#cases-ui-custom-fields[add a custom field] to a case, and set the field to be required, you cannot leave the field empty. You must set a default value for it.
```
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/whats-new.asciidoc:138]: Yeah, the wording here is a little awk and assumes that users are already familiar with the [custom fields](https://www.elastic.co/guide/en/kibana/8.11/whats-new.html#_kibana_cases_custom_field) feature for cases. This feature essentially allows you to add your own custom fields to a case so you can further configure and categorize it. When you create a custom field, you set it as optional (users don't have to fill it out when creating a case) or required (users have to fill it out). The above summary describes custom fields that are set up to be required, and says that required fields must have a value. 

Maybe we can change the summary to this:

```suggestion
When adding {security-guide}/cases-open-manage.html#cases-ui-custom-fields[custom fields] to a case, any fields that are mandatory must have a default value.
```

Here's another, more direct option:

```suggestion
Mandatory {security-guide}/cases-open-manage.html#cases-ui-custom-fields[custom case fields] must have a default value.
```

[the remark this answers is not in the document]
Comment (jmikell821) [reply docs/whats-new.asciidoc:138]:
```suggestion
When adding {security-guide}/cases-open-manage.html#cases-ui-custom-fields[custom fields] to a case, any mandatory fields must have a default value.
```

**Pull request merged**